src/clients/c/tb_client.zig

```diff
@@ -49,6 +53,77 @@ const TestingContext = blk: {
     break :blk ContextType(EchoClient);
 };
 
+/// Logging is global per application; it would be nice to be able to define a different logger for
+/// each client instance, though.
```

Comment:
I don't think there's an easy way to do this, but I'd love to be wrong!

Reply:
Yes, Zig's logging infrastructure has no context, just a global function pointer. We could rely on the runtime specifics to store the context (such as napi/jni), but not for shared/static libs!

While it'd be really nice to isolate logging _per client_, the downside is that we wouldn't be able to log situations where the context isn't yet available (like, for example, creating the client itself).

So, I don't think it's worth trying, a global logger is good enough. Most importantly, when we wire this into the clients, we should consistently use this same logger to output any log emitted by the language side.